templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,8 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- filenames = get_files.map {|f| f.path} -%>
+<%- configfilenames = get_configfiles.map {|cf| cf.path} -%>
+<%- (dirnames + filenames + configfilenames).each do |name| -%>
+Copy-Item (Join-Path $toolsDir '<%= name %>') <%= platform.convert_to_windows_path(name) %> -Force
```

Comment:
So, we have a list of files when we're building the package, stored in a text file called `file-list`. Would it be worthwhile to parse these files in the chocolateyInstall and chocolateyUninstall scripts when we copy them into the tools directory to we know exactly what is getting installed and removed? I'm more concerned about this for the uninstall script, so that we only uninstall boost files when boost is uninstalled, and not yaml-cpp files, even though they're both be installed under `C:\tools\pl-build\tools`